tests/integration/components/identity-steps/step-seven-test.js

```diff
@@ -0,0 +1,161 @@
+  });
+
+  test('render description on verification page when profile status is pending', async function (assert) {
+    const objToCheckFunction = {
+      isHandleRefresh: assert.ok(true, 'handleRefresh is working fine!'),
+    };
+    this.set('handleRefresh', () => {
+      objToCheckFunction.isHandleRefresh;
+    });
+    await render(
+      hbs`<IdentitySteps::StepSeven @handleRefresh={{this.handleRefresh}} />`
+    );
+
+    assert
+      .dom('[data-test-verification-description-container]')
+      .hasClass('verification-page__description');
+    assert
+      .dom('[data-test-verification-description-1]')
+      .hasText('Refresh to Check Verification Status');
+    assert
+      .dom('[data-test-verification-description-2]')
+      .hasText('Your Profile Service Linked with Real Dev Squad Service');
+  });
+
+  test('render Refresh button on verification page when profile status is pending', async function (assert) {
+    const objToCheckFunction = {
+      isHandleRefresh: assert.ok(true, 'handleRefresh is working fine!'),
+    };
+    this.set('handleRefresh', () => {
+      objToCheckFunction.isHandleRefresh;
+    });
+    await render(
+      hbs`<IdentitySteps::StepSeven @handleRefresh={{this.handleRefresh}} />`
+    );
+
+    assert.dom('[data-test-button=refresh]').hasText('Refresh');
+    assert.dom('[data-test-button=refresh]').hasProperty('type', 'button');
+  });
+
+  test('clicking Refresh button refresh the verification page when profile status is pending', async function (assert) {
+    const objToCheckFunction = {
+      isHandleRefresh: assert.ok(true, 'handleRefresh is working fine!'),
+    };
+    this.set('handleRefresh', () => {
+      objToCheckFunction.isHandleRefresh;
+    });
+    await render(
+      hbs`<IdentitySteps::StepSeven @handleRefresh={{this.handleRefresh}} />`
+    );
+
+    await click('[data-test-button=refresh]');
+    assert.dom('[data-test-button=refresh]').hasText('Refresh');
+    assert.dom('[data-test-button=refresh]').hasProperty('type', 'button');
+  });
+
+  test('renders heading on verification page when profile status is blocked', async function (assert) {
+    const objToCheckFunctions = {
+      isGoToGenerateChaincodePageWorks: false,
+    };
+    this.set('goToGenerateChaincodePage', () => {
+      objToCheckFunctions.isGoToGenerateChaincodePageWorks;
+    });
+    this.loginService = this.owner.lookup('service:login');
+    this.set('loginService.userData.profileStatus', 'BLOCKED');
+    await render(
+      hbs`<IdentitySteps::StepSeven 
+        @goToGenerateChaincodePage={{this.goToGenerateChaincodePage}} 
+      />`
+    );
+
+    assert
+      .dom('[data-test-verification-heading]')
+      .hasClass('verification-page__heading');
+    assert.dom('[data-test-verification-heading]').hasText('Blocked');
+    assert.false(
+      objToCheckFunctions.isGoToGenerateChaincodePageWorks,
+      'goToGenerateChaincodePageWorks is working fine!'
+    );
```

Comment:
here you're basically checking `false === false`? Is it needed?
If you want to check function you have to perform the event in which handler is attached to

Reply:
Done with changes...can you please check latest comment @satyam73